File: r_csv/cars.R
library(rvest)

data <- read_html("https://en.wikipedia.org/wiki/Comma-separated_values")

tables <- data %>% html_table(fill = TRUE)

tables1 <- tables[[2]]

setwd("D:/YEAR 3/PROGRAMMING FOR DATA SCIENCE/Practice Assignments/st_2195_assignment_2/r_csv")

write.csv(tables1, "cars.csv")

tables2 <- read.csv("cars.csv")
data.frame(tables2)

File: python_csv/cars.py
import pandas as pd
import os
table = pd.read_html('https://en.wikipedia.org/wiki/Comma-separated_values')

df = table[1]
print(df)

os.chdir('D:\YEAR 3\PROGRAMMING FOR DATA SCIENCE\Practice Assignments\st_2195_assignment_2\python_csv')

df.to_csv(r'cars.csv', index = False, header = True)

df2 = pd.read_csv('cars.csv')
df2
